ckan/requirements.txt

```diff
@@ -4,19 +4,19 @@ Babel==2.10.3
 Beaker==1.11.0
 bleach==5.0.1
 blinker==1.5
-boto3==1.35.12
-botocore==1.35.12
+boto3==1.35.49
+botocore==1.35.49
 certifi==2024.8.30
-cffi==1.17.0
+cffi==1.17.1
 chardet==5.2.0
-charset-normalizer==3.3.2
+charset-normalizer==3.4.0
 ckan @ git+https://github.com/GSA/ckan.git@8c4a517efeac80db098cc6ba144cb742bbeca194
 -e git+https://github.com/ckan/ckanext-archiver.git@cbfadf9fbf10405958fdef9f77a7faedc05aa20b#egg=ckanext_archiver
-ckanext-datagovcatalog==0.1.0
+ckanext-datagovcatalog==0.1.1
```

Comment:
build complaining about this

Reply:
can you approve https://github.com/GSA/ckanext-datagovcatalog/pull/41 first?